packages/cosmic-swingset/src/kernel-stats.js

```diff
@@ -256,9 +256,14 @@ export function makeInboundQueueMetrics(initialLength) {
   let remove = 0;
 
   return harden({
-    incStat: (delta = 1) => {
-      length += delta;
-      add += delta;
```

Comment:
We're still calling `decStat` when picking the elements one by one, so that the value of the stat is reflected immediately. I could replace the `decStat` calls by an `updateLength(actionQueue.size())` but probing the queue head/tail felt inappropriate (even though those values are cached in memory) when we know exactly how we modified the queue length.